phpunit.xml.dist

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- http://phpunit.de/manual/4.1/en/appendixes.configuration.html -->
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/4.1/phpunit.xsd"
+         backupGlobals="false"
+         colors="true"
+         bootstrap="app/bootstrap.php.cache"
+>
+    <testsuites>
+        <testsuite name="Project Test Suite">
+            <directory>src/*/*Bundle/Tests</directory>
```

Comment:
Do we have any directory that matches this pattern? Shouldn't it be `src/*/Component/*/Tests` as the component equivalent of the bundle one line below?